{{cookiecutter.project_slug}}/dev.yml

```diff
@@ -19,7 +19,8 @@ services:
       dockerfile: ./compose/django/Dockerfile-dev
     command: /start-dev.sh
     depends_on:
-      - postgres
+      - postgres{% if cookiecutter.use_mailhog == 'y' %}
```

Comment:
Thank you for clarifying! I'll explore the options in more detail, and alter the PR accordingly.